src/Enum.php

```diff
@@ -41,7 +41,7 @@
     public function __construct($value)
     {
         if (!$this->isValid($value)) {
-            throw new \UnexpectedValueException("Value '$value' is not part of the enum " . get_called_class());
+            throw new \UnexpectedValueException("Value '$value' is not part of the enum " . \get_called_class());
```

Comment:
I don't think it's necessary to optimize for error cases.

Reply:
Thanks. I've simply put a backslash before _all_ function calls.

Reply:
OK let's leave it that way. You're right it's no big deal.